doc_src/en/OmegaT5_ProjectFolder.xml

```diff
@@ -278,15 +285,25 @@
 
 		  <para>Translations from TMX files placed in this folder are
 		  automatically inserted in matching segments without the automatic
-		  prefix, making it unnecessary to confirm these segments. Translations
-		  coming from this folder are considered to be <emphasis>more</emphasis>
-		  reliable than the current project memory.</para>
+		  prefix, making it unnecessary to confirm these segments.</para>
```

Comment:
I think either "without the fuzzy match prefix" or "without any prefix" would be better than "without the automatic prefix".